### DISCUSSION TITLE
When is the next release?

### DISCUSSION THREAD

**Discussion by gerroon** [JacquesLucke/animation_nodes#1718] [General]
Hi

Is there a time line for the next release? Blender 2.93 is Python 3.9 and Animation Nodes is complaining about it.

**Comment by OmarEmaraDev** [comment]
We will probably align ourselves with the 2.93 LTS release.

**Reply by gerroon** [reply]
Ok thanks, so we just need to wait for the actual Blender release first.

**Comment by gerroon** [comment]
Hi

2.93 is out. Do you know when you will be releasing it?

**Reply by OmarEmaraDev** [reply]
@gerroon  Apologies for the delay. I am not currently active because of my final exams. So we had to delay the release until I get back to it. We should be able to start preparing by the end of this month.

**Comment by gerroon** [comment]
Hi

Sorry to bug you again. Is there an ETA for this?

**Reply by OmarEmaraDev** [reply]
We already started on the release process, currently updating the documentation. You can track the progress in #1747.
